{
  "document_title": "CVE-2015-0220",
  "document_type": "SUSE CVE",
  "document_publisher": {
    "type": "Vendor",
    "contact_details": "security@suse.de",
    "issuing_authority": "SUSE Security Team"
  },
  "document_tracking": {
    "identification": {
      "id": "SUSE CVE-2015-0220"
    },
    "status": "Interim",
    "version": "1",
    "revision_history": {
      "revision": {
        "number": "8",
        "date": "2022-10-15T17:08:38Z",
        "description": "current"
      }
    },
    "initial_release_date": "2021-05-30T13:25:45Z",
    "current_release_date": "2022-10-15T17:08:38Z",
    "generator": {
      "engine": "cve-database/bin/generate-cvrf-cve.pl",
      "date": "2020-12-27T01:00:00Z"
    }
  },
  "document_notes": [
    {
      "text": "CVE-2015-0220",
      "title": "CVE",
      "type": "Summary"
    },
    {
      "text": "The django.util.http.is_safe_url function in Django before 1.4.18, 1.6.x before 1.6.10, and 1.7.x before 1.7.3 does not properly handle leading whitespaces, which allows remote attackers to conduct cross-site scripting (XSS) attacks via a crafted URL, related to redirect URLs, as demonstrated by a \"\\njavascript:\" URL.",
      "title": "Mitre CVE Description",
      "type": "Description"
    },
    {
      "text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
      "title": "Terms of Use",
      "type": "Legal Disclaimer"
    }
  ],
  "document_references": [
    {
      "type": "Self",
      "url": "https://lists.suse.com/pipermail/sle-security-updates/2015-March/001310.html",
      "description": "E-Mail link for SUSE-SU-2015:0563-1"
    },
    {
      "type": "Self",
      "url": "https://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html",
      "description": "E-Mail link for openSUSE-SU-2015:1598-1"
    },
    {
      "type": "Self",
      "url": "https://www.suse.com/support/security/rating/",
      "description": "SUSE Security Ratings"
    }
  ],
  "product_tree": {
    "branch": [
      {
        "type": "Product Family",
        "name": "SUSE OpenStack Cloud 4",
        "branch": [
          {
            "type": "Product Name",
            "name": "SUSE OpenStack Cloud 4",
            "full_product_name": {
              "text": "SUSE OpenStack Cloud 4",
              "product_id": "SUSE OpenStack Cloud 4",
              "cpe": "cpe:/o:suse:cloud:4"
            }
          }
        ],
        "full_product_name": {}
      },
      {
        "type": "Product Version",
        "name": "python-django-1.5.12-0.7.1",
        "full_product_name": {
          "text": "python-django-1.5.12-0.7.1",
          "product_id": "python-django-1.5.12-0.7.1"
        }
      }
    ],
    "relationship": [
      {
        "product_reference": "python-django-1.5.12-0.7.1",
        "relation_type": "Default Component Of",
        "relates_to_product_reference": "SUSE OpenStack Cloud 4"
      }
    ]
  },
  "vulnerability": {
    "notes": [
      {
        "text": "The django.util.http.is_safe_url function in Django before 1.4.18, 1.6.x before 1.6.10, and 1.7.x before 1.7.3 does not properly handle leading whitespaces, which allows remote attackers to conduct cross-site scripting (XSS) attacks via a crafted URL, related to redirect URLs, as demonstrated by a \"\\njavascript:\" URL.",
        "title": "Vulnerability Description",
        "type": "General"
      }
    ],
    "cve": "CVE-2015-0220",
    "product_statuses": [
      {
        "type": "Fixed",
        "product_id": [
          "SUSE OpenStack Cloud 4:python-django-1.5.12-0.7.1"
        ]
      }
    ],
    "threats": [
      {
        "type": "Impact",
        "description": "moderate"
      }
    ],
    "cvss_score_sets": {
      "score_set_v_2": {
        "base_score_v_2": "4.3",
        "vector_v_2": "AV:N/AC:M/Au:N/C:N/I:P/A:N"
      },
      "score_set_v_3": {}
    }
  }
}
